ansible/configs/ocp4-cluster/default_vars.yml

```diff
@@ -91,19 +91,31 @@ worker_instance_count: 2
 
 # Install OpenShift 4 - and which version
 install_ocp4: true
-ocp4_installer_version: "4.5.15"
+
+# Use 4.9 for latest available release in the `stable-4.9` directory.
+# Use 4.9.9 for exactly the specified release
+ocp4_installer_version: "4.9"
+
 # Run logic to enable cluster shutdown before 24h initial certificate rotation
 # Only works for OCP 4.1 and 4.2. OCP 4.4.8 and later no longer require this.
 ocp4_enable_cluster_shutdown: false
 
 # Where to download the OpenShift installer and client binaries from
 # Only used if the ocp4_installer_url and ocp4_client_url are not defined
 # Official Mirror
-# ocp4_installer_root_url: https://mirror.openshift.com/pub/openshift-v4/clients
+# ocp4_installer_root_url: >-
+#   {{ 'https://mirror.openshift.com/pub/openshift-v4/aarch64/clients' if openshift_architecture | default('x86_64') is match('arm64')
+#   else 'https://mirror.openshift.com/pub/openshift-v4/clients'
+#   }}
+
 # CloudFront Mirror
-ocp4_installer_root_url: http://d3s3zqyaz8cp2d.cloudfront.net/pub/openshift-v4/clients
+ocp4_installer_root_url: >-
+  {{ 'http://d3s3zqyaz8cp2d.cloudfront.net/pub/openshift-v4/aarch64/clients' if openshift_architecture | default('x86_64') is match('arm64')
+  else 'http://d3s3zqyaz8cp2d.cloudfront.net/pub/openshift-v4/clients'
+  }}
```

Comment:
Why is the default for `ocp4_installer_root_url` in the config defaults? Shouldn't this value be in the `host-ocp4-installer` role defaults?

Reply:
It's always been that way...  maybe something to tackle on AD 2.0?